pom.xml

```diff
@@ -900,8 +900,8 @@
                         <phase>validate</phase>
                         <configuration>
                             <configLocation>.checkstyle/checkstyle.xml</configLocation>
-                            <headerLocation>.checkstyle/java.header</headerLocation>
```

Comment:
Isn't this something from your check style PR?

Reply:
Yes yes... I based this PR on that branch so I will remove this one... :) Basically, this is only a different line..but I will update it.